samples/graph-notification-team-channel/csharp/GraphNotification/Provider/GraphBetaClient.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ChannelNotification.Provider
+{
+    using Azure.Identity;
+    using ChannelNotification.Model.Configuration;
+    using Microsoft.Extensions.Options;
+    using Microsoft.Graph;
+
+    public class GraphBetaClient
```

Comment:
I am not sure why the class name contains Beta

Reply:
I will check and update.